test/resources/statuschecker.py

```diff
@@ -177,14 +177,14 @@ def _get_log_message(self, msg_str):
     import os
 
     if not 2 <= len(sys.argv) <= 3 or '--help' in sys.argv:
-        print __doc__
+        print(__doc__)
```

Comment:
The whole `statuschecker.py` should actually be removed in favor of https://github.com/robotframework/statuschecker.